doc/source/ray-core/monitoring-debugging/getting-help.rst

```diff
@@ -0,0 +1,72 @@
+.. _ray-troubleshoot-getting-help:
+
+Getting Help
+============
+
+Ask Ray community or navigate through the documentation if your problem is not solved by the guide above.
+
+Getting Help from the Documentation
+-----------------------------------
+If you are stuck, navigate through the ray documentation. It is structured to answer a wide range of questions.
+
+- :ref:`Key Concepts <core-key-concepts>` explains key concepts of Ray.
+- :ref:`Deployment Guide <ref-deployment-guide>` explains you how to deploy Ray clusters and submit code to them.
+- :ref:`Ray Core User Guide <core-use-guide>` explains how to use Ray task, actor, object, placement groups, and dependencies.
+- :ref:`Task Design Pattern <task-patterns>`, :ref:`Actor Design Pattern <actor-patterns>`, and :ref:`Ray Common Gotchas <gotchas>` explains best practice, anti-pattern, and common gotchas of Ray.
```

Comment:
We have merged these into one patterns and anti-patterns section

Reply:
<img width="328" alt="Screen Shot 2022-10-04 at 8 01 02 AM" src="https://user-images.githubusercontent.com/18510752/193854482-3d31575e-4c6d-4dd8-9de9-fb377a48e3f8.png">

Actually I think we still has this. I will add a new link to pattern/antipattern section

Reply:
Yea, I'm in the middle of moving. By 2.1, everything should be moved to the new section.